src/Files.Backend/Files.Backend.csproj

```diff
@@ -1,22 +1,22 @@
 ﻿<Project Sdk="Microsoft.NET.Sdk">
 
   <PropertyGroup>
-    <TargetFramework>net7.0</TargetFramework>
-    <Nullable>enable</Nullable>
-    <IsTrimmable>true</IsTrimmable>
-    <Configurations>Debug;Release;Sideload</Configurations>
-    <Platforms>x86;x64;arm64</Platforms>
-    <RuntimeIdentifiers>win10-x86;win10-x64;win10-arm64</RuntimeIdentifiers>
+	<TargetFramework>net7.0</TargetFramework>
+	<Nullable>enable</Nullable>
+	<IsTrimmable>true</IsTrimmable>
+	<Configurations>Debug;Release;Sideload</Configurations>
+	<Platforms>x86;x64;arm64</Platforms>
+	<RuntimeIdentifiers>win10-x86;win10-x64;win10-arm64</RuntimeIdentifiers>
   </PropertyGroup>
 
   <ItemGroup>
-    <PackageReference Include="ByteSize" Version="2.1.1" />
-    <PackageReference Include="CommunityToolkit.Mvvm" Version="8.0.0" />
+	<PackageReference Include="ByteSize" Version="2.1.1" />
+	<PackageReference Include="CommunityToolkit.Mvvm" Version="8.0.0" />
   </ItemGroup>
 
   <ItemGroup>
-    <ProjectReference Include="..\Files.Sdk.Storage\Files.Sdk.Storage.csproj" />
-    <ProjectReference Include="..\Files.Shared\Files.Shared.csproj" />
+	<ProjectReference Include="..\Files.Sdk.Storage\Files.Sdk.Storage.csproj" />
+	<ProjectReference Include="..\Files.Shared\Files.Shared.csproj" />
   </ItemGroup>
```

Comment:
`PropertyGroup` and `ItemGroup` have inconsistant formatting with spaces instead of tabs if you change the formatting of the others attributes. 
Or should we use only space in the `.csproj` as in the source code?

Reply:
This should be fixed now